raiden/tests/unit/transfer/mediated_transfer/test_mediation_fee.py

```diff
@@ -68,27 +68,31 @@ def test_imbalance_penalty():
             (TokenAmount(100), FeeAmount(10)),
         ]
     )
-    assert v_schedule.fee_payer(
+    assert v_schedule.fee_payee(
         balance=Balance(100 - 0), amount=PaymentWithFeeAmount(50)
     ) == FeeAmount(-10)
+    # The payer channel works in the opposite direction
     assert v_schedule.fee_payer(
+        balance=Balance(100 - 0), amount=PaymentWithFeeAmount(-50)
```

Comment:
Shouldn't this throw? The channel has a total balance of 100, we have 100 balance and want to receive a payment of 50, for which the partner has no balance.

Reply:
This assert was a bad idea. The amount should not be negative at this point (I was thinking about using a negative amount to read from the IP function). I'll change the test to something same.

Reply:
The test should make more sense, now.

Reply:
Thanks! That's a much nicer test!